contents/blog/posthog-vs-amplitude.mdx

```diff
@@ -115,30 +97,34 @@ PostHog's offers a generous free 1 million A/B testing events. Amplitude's A/B t
-  <ComparisonRow column1={true} column2={false} feature="Web app recordings" description="Capture recordings from single-page apps" />
-  <ComparisonRow column1="Beta" column2={false} feature="Mobile app recordings" description="Capture recordings in iOS and Android apps" />
-  <ComparisonRow column1={true} column2={false} feature="Identity detection" description="Link recordings to user IDs" />
+  <ComparisonRow column1="5,000 recordings" column2="1,000 recordings" feature="Free usage" description="How much free usage do you get each month?" />
+  <ComparisonRow column1={true} column2={true} feature="Web app recordings" description="Capture recordings from single-page apps" />
+  <ComparisonRow column1={true} column2={false} feature="Mobile app recordings" description="Capture recordings in iOS and Android apps" />
+  <ComparisonRow column1={true} column2={false} feature="Heatmaps" description="Visualize where users click in your app or website" />
+  <ComparisonRow column1={true} column2={true} feature="Identity detection" description="Link recordings to user IDs" />
   <ComparisonRow column1={true} column2={false} feature="Console logs" description="Capture extra content from a user's browser" />
   <ComparisonRow column1={true} column2={false} feature="Playlists" description="Sort recordings into static and dynamic playlists" />
   <ComparisonRow column1={true} column2={false} feature="Performance monitoring" description="Track network events within a session" />
-  <ComparisonRow column1={true} column2={false} feature="Privacy masking" description="Censor personal information from playback" />
+  <ComparisonRow column1={true} column2={true} feature="Privacy masking" description="Censor personal information from playback" />
   <ComparisonRow column1={true} column2={false} feature="Conditional recording" description="Only capture the sessions you want" />
   <ComparisonRow column1={true} column2={false} feature="DOM explorer" description="Explore an interactive snapshot of replays" />
   <ComparisonRow column1={true} column2={false} feature="Export recordings" description="Save important recordings offline" />
-  <ComparisonRow column1={true} column2={false} feature="Minimum duration" description="Only record sessions longer than the minimum duration" />
   <ComparisonRow column1={true} column2={false} feature="Sample recorded sessions" description="Restrict the percentage of sessions that will be recorded" />
   <ComparisonRow column1={true} column2={false} feature="Record via feature flag" description="Only record sessions for users that have the flag enabled" />
 </ComparisonTable>
 
-### Surveys
+> 💡 **Good to know:** Replays let you watch how users experience your app, diagnose issues, improve support, and understand real user behavior in a way raw data can't. They are reconstructions of the session, _not_ video recordings of user's screen. Private info, such as passwords, is masked.
```

Comment:
I think it should technically be "is" but I think "are" is better after reading it after "passwords"
```suggestion
> 💡 **Good to know:** Replays let you watch how users experience your app, diagnose issues, improve support, and understand real user behavior in a way raw data can't. They are reconstructions of the session, _not_ video recordings of user's screen. Private info, such as passwords, are masked.
```